api-reference/10 UI Widgets/GridBase/3 Methods/navigateToRow(key).md

```diff
@@ -3,7 +3,7 @@ id: GridBase.navigateToRow(key)
 ---
 ---
 ##### shortDescription
-Navigates the grid to the data page that contains the row with the specified key and scrolls the grid to display the row if it is not in the viewport.
+Navigates the grid to the data page that contains the row with the specified key, expand groups if necessary, and scrolls the grid to display the row if it is outside the viewport.
```

Comment:
"expand groups if necessary" Why 'if  necessary'?

Reply:
Because groups will be expanded only if data should be grouped and the group with the target row is collapsed. If there's no grouping or the group is already expanded, then the method won't expand it.

Reply:
But you cannot state 'if necessary' because this is not some necessity. If something happens in certain cases, then you should mention these cases, not that they happen when it is necessary.